clusterman/cli/toggle.py

```diff
@@ -39,6 +42,17 @@ def disable(args: argparse.Namespace) -> None:
         Item=state,
     )
 
+    time.sleep(1)  # Give DynamoDB some time to settle
+    if not autoscaling_is_paused(args.cluster, args.pool, args.scheduler, arrow.now()):
+        print('Something went wrong!  The autoscaler is NOT paused')
+    else:
+        now_str = str(parse_time_string('now').to('local'))
+        until_str = str(parse_time_string(args.until).to('local'))
```

Comment:
If `args.util` is None, `parse_time_string` will raise an `AttributeError` on [this](https://github.com/Yelp/clusterman/blob/master/clusterman/util.py#L176) line because it does not have a `split` function. Should probably just put it under the `args.until` conditional you have later in the function.

Reply:
whoops, good catch.